src/it/java/com/google/checkstyle/test/chapter5naming/rule528typevariablenames/MethodTypeParameterNameTest.java

```diff
@@ -21,8 +21,8 @@
 
 import java.util.Map;
 
-import org.junit.BeforeClass;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeAll;
```

Comment:
same as previous and other classes.

Reply:
Done. Now we have two import control configs: one for `com.google/sun` another for `org.checkstyle`

Reply:
@pbludov I'm sorry, I didn't notice this was integration only changes. I don't think we need 2 import control configs.
If we are doing this in groups in separate PR (tests, int1, int2, etc..), we can postpone the import control or add a suppression for areas not done yet. @romani likes the suppression more so as it prevents others from undoing what you have already started.

Reply:
Done. Actually, once this issue will be done, all `junit4` packages will not be accessible at all as the reference to the module `junit4` will be removed from the `pom.xml` file.